forge/forge/tvm_to_python.py

```diff
@@ -696,7 +696,14 @@ def populate_conv2d_transpose_args(graph, nid, compiler_cfg):
         )
     )
 
-    in_channel = next((n["attrs"]["shape"][0][0][0] for n in graph["nodes"] if n["name"] == "model.weight"), None)
+    in_channel = None
+    for input_ in node["inputs"]:
+        input_nid = input_[0]
+        input_node = graph["nodes"][input_nid]
+        if input_node["op"] == "parameter" and input_node["name"].endswith("weight"):
+            in_channel = input_node["attrs"]["shape"][0][0][0]
```

Comment:
How does it work for channel first and channel last cases?

Reply:
input_node["attrs"]["shape"][0][0][0] corresponds to the in-channels from the param being sent
In case of below example, it will be 16 and it would be the same for channelfirst and channel last cases
```
"in_channels, out_channels, kernel_size, stride, padding, groups, bias, dilation, padding_mode, input_shape",
    [ 16, 33, (3, 3), 2, 0, 1, True, 1, "zeros", (20, 16, 50, 100)) ]
```